### PR TITLE
Add security policy

### PR DESCRIPTION
**Pull request by sethvargo**
Relatively new GitHub thing: https://help.github.com/en/github/managing-security-vulnerabilities/adding-a-security-policy-to-your-repository

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+To report a suspected vulnerability, please contact
+exposure-notifications-feedback@google.com and included the steps to
```
Comment (mikehelmick) [inline .github/SECURITY.md:4]: s/included/include/

Comment (sethvargo) [reply .github/SECURITY.md]: Updated

**Pull request merged**